test/lib/util/aws/cloudfront_signed_url_test.exs

```diff
@@ -0,0 +1,96 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
+
+defmodule Antikythera.Aws.CloudfrontSignedUrlTest do
+  use Croma.TestCase
+  alias Antikythera.Aws.CloudfrontSignedUrl
```

Comment:
This alias is unnecessary.

Reply:
I fixed in https://github.com/access-company/antikythera/pull/100/commits/79611166a960d8f3a744335322284bc087d5db55